TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/tracing/KafkaConsumerContexts.java

```diff
@@ -59,7 +59,7 @@ private MetricInstruments(Meter meter) {
 
         public void onPartitionsRevoked(Collection<TopicPartition> partitions) {
             meterIncrementEvent(getMetrics().kafkaPartitionsRevokedCounter);
-            onParitionsAssignedChanged(partitions.size());
+            onParitionsAssignedChanged(-1 * partitions.size());
```

Comment:
Could you fix the spelling of this function? M

Reply:
Addressed